tests/components/application-connector/resources/charts/application-connectivity-validator-test/templates/test.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: application-connectivity-validator-test
+  namespace: {{ .Values.namespace }}
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      annotations:
+        traffic.sidecar.istio.io/excludeOutboundPorts: "8080"
+    spec:
+      serviceAccountName: connectivity-validator-test
+      initContainers:
+        - name: wait-for-services
```

Comment:
Before the merge we most likely must update docker images to be pulled from kyma external registry (not from docker.io)

Reply:
For now we decided to leave it as it is.